src/api.js

```diff
@@ -13,18 +13,22 @@ let _baseUrl;
 const setBaseUrl = (baseUrl) => _baseUrl = baseUrl;
 const getUrl = (fragment) => normalizeUrl(`${_baseUrl}/${fragment}`);
 const getHeaders = (authToken) => ({
-	Authorization: `Bearer ${authToken}`
+	Authorization: `Bearer ${authToken}`,
+	'Content-Type': 'application/json'
```

Comment:
Content-Type is not necessary when you set a `json` property, see https://github.com/sindresorhus/got/blob/a3da70a78aeb7f44dd3e0d0fa47cebe9541eb91e/source/core/index.ts#L442

Reply:
It could be a problem with our API, if it requires Content-Type even when there is no body